### PR TITLE
add: added client-cloud-services

### PR DESCRIPTION
**Pull request by nevil-mathew**

### CODE REVIEW

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -52,88 +40,111 @@ services:
...
       - elevate_net
+    deploy:
+      resources:
+        limits:
+          cpus: '4.0' # Set CPU limit to 4 cores
+          memory: '200M' # Set memory limit to 2 GB
   notification:
     build: '../notification/'
-    image: elevate/notification:1.0
+    image: elevate/notification:2.4
```
Comment (rakeshSgr) [inline docker-compose-dev.yml:91]: @nevil-mathew its refer to local image

File: src/.env.sample [hunk 2]
```diff
@@ -182,3 +182,21 @@ MINIMUM_DURATION_FOR_AVAILABILITY=15
 
 #True to allow multiple bookings
 MULTIPLE_BOOKING=false
+
+
+# Cloud storage provider in azure, aws, gcloud, oci or s3
+CLOUD_STORAGE_PROVIDER=aws
+# Cloud storage identity [Azure Account Name, AWS Access Key, GCP Client Email, OCI S3 Access Key or S3 Access Key]
+CLOUD_STORAGE_ACCOUNTNAME=AKIAIOSFODNN7EXAMPLE
```
Comment (rakeshSgr) [inline src/.env.sample:190]: @nevil-mathew use dummy values

File: src/.env.sample [hunk 3]
```diff
@@ -182,3 +182,21 @@ MINIMUM_DURATION_FOR_AVAILABILITY=15
 
 #True to allow multiple bookings
 MULTIPLE_BOOKING=false
+
+
+# Cloud storage provider in azure, aws, gcloud, oci or s3
+CLOUD_STORAGE_PROVIDER=aws
```
Comment (rakeshSgr) [inline src/.env.sample:188]: @nevil-mathew remove old env's
DEFAULT_AZURE_CONTAINER_NAME

File: docker-compose-dev.yml (hunk 1 above)
Comment (nevilmathew) [reply docker-compose-dev.yml:91]: It's not the main docker compose file, it's untouched.
This docker compose is intended to be used for development.

Comment (nevil-mathew) [reply docker-compose-dev.yml:91]: This is not the main docker compose file, it is to be used for local development

File: src/.env.sample (hunk 2 above)
Comment (nevil-mathew) [reply src/.env.sample:190]: These are dummy values

**Pull request merged**